app/src/main/java/de/westnordost/streetcomplete/quests/barrier_opening/AddBarrierOpening.kt

```diff
@@ -39,7 +42,7 @@ class AddBarrierOpening(
         get() = if (!checkArSupport()) R.string.default_disabled_msg_no_ar else 0
 
     override fun getTitle(tags: Map<String, String>) =
-        if (tags["barrier"] == "bollard" || tags["barrier"] == "block") {
+        if (tags["barrier"] == "bollard" || tags["barrier"] == "block" || tags["cycle_barrier"] == "diagonal") {
```

Comment:
Why?

Reply:
For diagonal cycle barriers, the text _"What is the widest gap one can pass through here?"_ seemed  more appropriate than _"What is the width of the opening here?"_ to me. 

I can remove it if you prefer both have the same wording, of course?